.github/workflows/test-and-deploy-alpha.yml

```diff
@@ -33,67 +102,60 @@ jobs:
           customtypesapi_endpoint: ${{ secrets.CUSTOMTYPESAPI_ENDPOINT }}
           user_service_endpoint: ${{ secrets.USER_SERVICE_ENDPOINT }}
           acl_provider_endpoint: ${{ secrets.ACL_PROVIDER_ENDPOINT }}
-        run: npm run test
-  
-  dep-check:
-    needs: install-deps
-    runs-on: [ubuntu-latest]
-    steps: 
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
-      - uses: ./.github/actions/build-essential
-      - name: Dependency Checks
-        run: npm run depcheck
-  
-  prettier:
-    needs: install-deps
-    runs-on: [ubuntu-latest]
-    steps: 
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
-      - uses: ./.github/actions/build-essential
-      - name: Prettier
-        run: npm run prettier:check
-  
-  linting:
-    needs: install-deps
-    runs-on: [ubuntu-latest]
-    steps: 
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
-      - uses: ./.github/actions/build-essential
-      - name: Linting
-        run: lerna run lint
 
-  audit:
-    needs: install-deps
-    runs-on: [ubuntu-latest]
-    steps: 
-      - uses: actions/checkout@v2
-      - uses: ./.github/actions/restore-cache
-      - uses: ./.github/actions/setup-node
-      - uses: ./.github/actions/build-essential
-      - name: Audit
-        run: lerna run audit
+  packages-win32:
+    needs: prepare-win32
+    runs-on: [windows-latest]
+    strategy:
+      fail-fast: false
+      matrix:
+        package:
+          - packages/plugin-kit
+          - packages/manager
+          - packages/init
+          - packages/slice-machine
+          - packages/start-slicemachine
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Restore cache
+        uses: ./.github/actions/restore-cache
+
+      - name: Set up Node
+        uses: ./.github/actions/setup-node
+
+      - name: Unit
+        run: npm run unit --workspace ${{ matrix.package }}
+        env:
```

Comment:
do the unit tests nee env vars?

Reply:
As of today, it appears they don't, if they ever need some they can be added.